src/mixins/object_geometry.mixin.js

```diff
@@ -305,10 +305,8 @@
     setCoords: function() {
       var theta = degreesToRadians(this.angle),
           vpt = this.getViewportTransform(),
-          f = function (p) {
-            return fabric.util.transformPoint(p, vpt);
-          },
-          p = this._calculateCurrentDimensions(false),
+          f = fabric.util.transformPoint,
```

Comment:
I'm not a fan of 1-2 char names, even though I see we already had it! Can we rename it to at least `transform`?